README.md

```diff
@@ -0,0 +1,85 @@
+# ImageCarouselReact
+Author: Akileng Isaac, Flock of Birds
+
+Type: Widget
+Latest Version: 1.0
+Package file name: ImageCarouselReact.mpk
+
+## Description
+
+Displays images in a carousel either from the database or from the modeler
+
+## Typical usage scenario
+
+* Displays Images in a carousel
+
+## Features and limitations
+
+* Static and dynamic images (From modeler or the database)
+* Use Navigations controls to move images
+* Auto Slide of images based on an Interval
+* Onclick Event to Images to call Microflow or Open Page
+* Height and Width configurations
+
+## Installation
+
+See the general instructions under _Configuration._
+
+## Dependencies
+
+* Mendix 6 Environment. Tested on 6.8.0
+
+
+## Configuration
+* Create an entity that inherits from the System.image 
+  * Add String "Caption" attribute (Optional)
+  * Add String "Description" attribute (Optional)
+
+* For static Images, add the images directly to the widget from Source - Static Tab
+
+
+## Properties
+* Data Source
+  * Entity;The entity with the images to display
+  * Source;Type of Source could be Xpath, Microflow or Static
+  * Caption;Based on attribe, caption shown on image
+  * Description;Based on attribe, Description shown on image
+* Source - Xpath
+  * Constraint;Constraint to image entity
+* Source - Microflow
+  * Data Source Microflow;Returns images for carousel
+* Source - Static
+  * Images To Display
+     - Data Source
+        - Image Caption;caption shown on image
+        - Image Description;Description shown on image
+        - Image/Picture;image to display on carousel
+     - Behaviour
+        - On Click;what action to excute when image is clicked
+        - Call Microflow; Microflow to execute when image is clicked
```

Comment:
The On click specifies what event to execute  Do nothing, Show a page or Call a microflow.
The Call microflow(now changed  to Microflow) specifies what microflow to execute.
Open Page(now changed to Page) specifies the page to open